catalyst-toolbox/src/rewards/veterans.rs

```diff
@@ -77,13 +77,35 @@ fn calc_final_eligible_rankings(
         .collect()
 }
 
+fn calc_final_ranking_consensus_per_review(rankings: &[impl Borrow<VeteranRankingRow>]) -> Decimal {
+    let rankings_majority = Decimal::from(rankings.len()) / Decimal::from(2);
+    let ranks = rankings.iter().counts_by(|r| r.borrow().score());
+
+    match (ranks.get(&FilteredOut), ranks.get(&Excellent), ranks.get(&Good)) {
+        (Some(filtered_out), _, _) if Decimal::from(*filtered_out) >= rankings_majority => {
+            Decimal::from(*filtered_out) / Decimal::from(rankings.len())
+        }
+        (_, Some(excellent), _) if Decimal::from(*excellent) > rankings_majority => {
+            Decimal::from(*excellent) / Decimal::from(rankings.len())
+        }
+        (_, Some(excellent), Some(good)) => {
+            (Decimal::from(*excellent) + Decimal::from(*good)) / Decimal::from(rankings.len())
+        }
+        (_, _, Some(good)) => {
+           Decimal::from(*good) / Decimal::from(rankings.len())
+        }
+        _ => Decimal::ONE,
```

Comment:
not sure this arm can ever be reached and I'm a bit confused by the fact we're using 1 here as a value

Reply:
ok thanks! fixed